chain/chain/src/test_utils/kv_runtime.rs

```diff
@@ -706,13 +706,22 @@ impl EpochManagerAdapter for MockEpochManager {
         Ok(chunk_producers[index].account_id().clone())
     }
 
+    fn get_chunk_validator_assignments(
+        &self,
+        _epoch_id: &EpochId,
+        _shard_id: ShardId,
+        _height: BlockHeight,
+    ) -> Result<Arc<ChunkValidatorAssignments>, EpochError> {
+        Ok(Arc::new(vec![]))
+    }
+
     fn get_chunk_validators(
```

Comment:
(See comment below) If HashSet is a part of `ChunkValidatorAssignments`, then it will be a method of `ChunkValidatorAssignments`.